src/sentry/static/sentry/app/utils/discover/eventView.tsx

```diff
@@ -534,6 +534,24 @@ class EventView {
     return decodeColumnOrder(this.fields);
   }
 
+  getDays(): number {
+    const statsPeriod = decodeScalar(this.statsPeriod);
+    const start = decodeScalar(this.start);
+    const end = decodeScalar(this.end);
+
+    if (statsPeriod && statsPeriod.endsWith('d')) {
+      return parseInt(statsPeriod.slice(0, -1), 10);
+    } else if (statsPeriod && statsPeriod.endsWith('h')) {
+      return parseInt(statsPeriod.slice(0, -1), 10) / 24;
+    } else if (start && end) {
+      return (
+        (new Date(end).getTime() - new Date(start).getTime()) / (24 * 60 * 60 * 1000)
+      );
+    } else {
+      return 0;
+    }
```

Comment:
```suggestion
    }
    return 0;
```

nit: You don't need an else as all other branches returned